cmd/protoc-gen-docs/htmlGenerator.go

```diff
@@ -1111,6 +1113,27 @@ func getFieldBehavior(options *descriptor.FieldOptions) []annotations.FieldBehav
 	return s
 }
 
+func getRequiredFromComments(loc protomodel.LocationDescriptor) bool {
+	com := loc.GetLeadingComments()
+	if com == "" {
+		com = loc.GetTrailingComments()
+		if com == "" {
+			return false
+		}
+	}
+
+	if strings.Contains(com, "Required") {
```

Comment:
I believe I might have seen REQUIRED in some files.

Reply:
And what happens if someone happened to use Required: no? I tend to agree with John's comment is could be hard to infer from the comments.

Reply:
I check istio/api repo, there's no something like `Required: no`

we have three types:
1. https://github.com/istio/api/blob/95d2a2fe214417029ef85405cd1a0be6f958467d/mesh/v1alpha1/config.proto#L463
1. https://github.com/istio/api/blob/95d2a2fe214417029ef85405cd1a0be6f958467d/telemetry/v1alpha1/telemetry.proto#L370
1. https://github.com/istio/api/blob/5bf1a9b66f3cc8cbeba872cc38b99a5fc0986651/networking/v1alpha3/destination_rule.proto#L587

IMO, (3) is the best, but there's no lint for it, also there will be a huge change to correct all of them.
if use comment, I prefer the kubernetes' way https://github.com/istio/api/blob/0d6e2d45f301a0f7590749a58418b8b37beffff3/common-protos/k8s.io/apimachinery/pkg/apis/meta/v1beta1/generated.proto#L35 

@ericvn @howardjohn WDYT?